www/profile/inner.js

```diff
@@ -261,10 +285,13 @@ define([
         }
 
         var addCancel = function () {
-            var cancelButton = h('button.btn.btn-danger.cp-app-profile-friend-request', [
-                h('i.fa.fa-user-times'),
-                Messages.cancel
-            ]);
+            var cancelButton = h('button', 
+                { class: 'btn btn-danger cp-app-profile-friend-request', 'aria-labelledby': 'cp-cancel-button' }, 
+                [
+                    h('i', { class: 'fa fa-user-times', 'aria-hidden': 'true' }),
+                    h('span', { id: 'cp-cancel-button' }, Messages.contacts_remove || 'Cancel Request')
```

Comment:
Same about hyperscript and don't hesitate to break the attribute object into multiple lines when it contains multiples elements.

And I'm not sure about the translation key here. If I understand/remember the code correctly, this is a button to cancel a contact request that has not been accepted yet. It was changed from "cancel" to "contacts_remove" but I don't understand why.
You should also remove the hardcoded value here: it's adding unnecessary bytes to the file and if you do so when adding a new key, we may skip that translation key and never notice that it's missing until we test with a different language. There may be some places in the code that still have such hardcoded translation but we want to avoid/remove them with our new processes.